tiles/src/main/java/com/protomaps/basemap/feature/NaturalEarthDb.java

```diff
@@ -0,0 +1,90 @@
+package com.protomaps.basemap.feature;
+
+import com.onthegomap.planetiler.util.FileUtils;
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NaturalEarthDb {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(NaturalEarthDb.class);
+
+  static {
+    try {
+      Class.forName("org.sqlite.JDBC");
+    } catch (ClassNotFoundException e) {
+      throw new IllegalStateException("sqlite JDBC driver not found");
+    }
+  }
+
+  private Map<String, NePopulatedPlace> placesByWikidataId;
+
+  public record NePopulatedPlace(String name, String wikidataId, String minZoom, String populationRank) {};
+
+  private NaturalEarthDb(List<NePopulatedPlace> places) {
+    this.placesByWikidataId = places.stream().filter(place -> place.wikidataId != null)
+      .collect(Collectors.toMap(place -> place.wikidataId, place -> place, (s, a) -> s)); // TODO conflicts
```

Comment:
@nvkelso there are instances in ne_10m_populated_places of multiple rows with the same Wikidata ID. Which one should win?

Reply:
The one with larger population (`pop_max`) value?

Let me dig into that a bit...